dubbo-monitor/dubbo-monitor-default/src/main/java/org/apache/dubbo/monitor/dubbo/DubboMonitor.java

```diff
@@ -91,7 +91,13 @@ public void send() {
             // get statistics data
             Statistics statistics = entry.getKey();
             AtomicReference<long[]> reference = entry.getValue();
-            long[] numbers = reference.get();
+            long[] numbers;
+            long[] update = new long[LENGTH];
+            //
+            do {
+                numbers = reference.get();
+            } while (!reference.compareAndSet(numbers, update));

```

Comment:
I don't think simply replacing `numbers` with `update` is right. You should see how `collect(URL url)` update statistics first.

Reply:
Can you describe it clearly? I just modified the logic of the reset here, I don't know where there is a problem.